### ISSUE TITLE
ga_account_list() error message: API Data failed to parse 

### ISSUE THREAD

**Issue by palooney** [MarkEdmondson1234/googleAnalyticsR#88]
Hello Mark,

I got an error message using ```  ga_account_list()```:

```
Token exists.
Request: https://www.googleapis.com/analytics/v3/management/accountSummaries/
Error in eval(assertion, env) : 
  argument "json.column" is missing, with no default
Warning message:
In acc_sum() :
  API Data failed to parse.  Returning parsed from JSON content.
                    Use this to test against your data_parse_function.
```

which I run perfectly 3 hours ago. I tried all methods mentioned [here](https://github.com/MarkEdmondson1234/googleAnalyticsR/issues/41#issuecomment-308195669) but I still got same error message.

Here is my ```sessionInfo()```:
<img width="1192" alt="screen shot 2017-06-13 at 1 37 12 pm" src="https://user-images.githubusercontent.com/9058204/27098557-9d67acbe-503d-11e7-901d-6de142337c16.png">

The codes I used are:

```r
## auth from GA
ga_auth()
## get account info
my_account = ga_account_list()
```
BTW I got same message after I run ```options(googleAuthR.verbose = 2)```.

Thanks in advance!

**Comment by MarkEdmondson1234** [comment]
This is caused by the update to dplyr 0.7.0, as detailed in this issue https://github.com/MarkEdmondson1234/googleAnalyticsR/issues/87

For now the solution is to downgrade `dplyr`:

```r
devtools::install_version("dplyr", version = "0.5.0")
```

**Comment by palooney** [comment]
Hello Mark,
I tried to downgrade ```dplyr``` but I still got the same error message.

**Comment by MarkEdmondson1234** [comment]
Did you restart R before trying again? Could you also confirm your dplyr version in the packe list.

**Comment by palooney** [comment]
Yes I did. dplyr's version is 0.5.

**Comment by MarkEdmondson1234** [comment]
Could you see if you're doing anything different from the people on this thread  #87 - are you getting the same error as before?

**Comment by palooney** [comment]
I just called ```ga_auth()``` function at very beginning and I didn't use the ```validate``` function. All others are the same and I got the same error as before.

**Comment by MarkEdmondson1234** [comment]
Ok this is most likely due to where you are deploying it not having `dplyr 0.5.0` (shinyapps?).  It will need to wait for a fix at `tidyjson` or here

**Comment by palooney** [comment]
Thanks for help!

**Comment by MarkEdmondson1234** [comment]
Does it work now with latest GitHub version?

**Comment by palooney** [comment]
Yes it does. Thanks so much!

**Comment by Emon-ProCoder7** [comment]
Hello Mark,

*I am in trouble finding proper credential authorization for my ''RoogleVision" from Google. Here I'm providing the whole detail.
On my GCP account, I've:*

     - just added no scope, just the support email - in OAuth consent screen
     - On my Credentials - Just created the - OAuth 2.0 Client IDs. With javascript origin URI and Redirect URI with my rstudio local domain and shiny.io.

I saved the credentials.json on my current working directory. On my R console I used this code:

`library(googleAuthR)
gar_set_client(web_json = "credentials.json")`

The browser opens the app just fine, but for connecting google API it shows this error in console:

>No authorization yet in this session! 
> No .httr-OAuth file exists in current working directory. Do library authentication steps to provide credentials.
 Warning: Error in : Invalid token 
78: stop 
77: simpleCall 
76: getGoogleVisionResponse 
72: observeEventHandler [/cloud/project/server.R#28]


Here is my session info:

>Failed to create bus connection: No such file or directory
─ Session info ─────────────────────────────────────────────────────────────────────────────
 setting  value                       
 version  R version 3.6.3 (2020-02-29)
 os       Ubuntu 16.04.6 LTS          
 system   x86_64, linux-gnu           
 ui       RStudio                     
 language (EN)                        
 collate  C.UTF-8                     
 ctype    C.UTF-8                     
 tz       Etc/UTC                     
 date     2020-07-10                  

>─ Packages ─────────────────────────────────────────────────────────────────────────────────
 package         * version  date       lib source                                   
 abind             1.4-5    2016-07-21 [1] RSPM (R 3.6.0)                           
 assertthat        0.2.1    2019-03-21 [1] RSPM (R 3.6.0)                           
 backports         1.1.8    2020-06-17 [1] RSPM (R 3.6.0)                           
 base64enc         0.1-3    2015-07-28 [1] RSPM (R 3.6.0)                                 
 bitops            1.0-6    2013-08-17 [1] RSPM (R 3.6.0)                           
 broom             0.5.3    2019-12-14 [1] RSPM (R 3.6.0)                           
 callr             3.4.3    2020-03-28 [1] RSPM (R 3.6.0)                           
 cellranger        1.1.0    2016-07-27 [1] RSPM (R 3.6.0)                           
 cli               2.0.2    2020-02-28 [1] RSPM (R 3.6.0)                           
 colorspace        1.4-2    2019-12-29 [1] R-Forge (R 3.6.0)                        
 crayon            1.3.4    2017-09-16 [1] RSPM (R 3.6.0)                           
 crosstalk         1.1.0.1  2020-03-13 [1] RSPM (R 3.6.0)                           
 DBI               1.1.0    2019-12-15 [1] RSPM (R 3.6.0)                           
 dbplyr            1.4.2    2019-06-17 [1] RSPM (R 3.6.0)                           
 desc              1.2.0    2018-05-01 [1] RSPM (R 3.6.0)                           
 devtools        * 2.3.0    2020-04-10 [1] RSPM (R 3.6.0)                           
 digest            0.6.25   2020-02-23 [1] RSPM (R 3.6.0)                           
 dplyr           * 0.8.3    2019-07-04 [1] RSPM (R 3.6.0)                           
 DT              * 0.13     2020-03-23 [1] RSPM (R 3.6.0)                           
 EBImage         * 4.28.1   2019-12-06 [1] Bioconductor                             
 ellipsis          0.3.1    2020-05-15 [1] RSPM (R 3.6.0)                           
 evaluate          0.14     2019-05-28 [1] RSPM (R 3.6.0)                           
 fansi             0.4.1    2020-01-08 [1] RSPM (R 3.6.0)                           
 fastmap           1.0.1    2019-10-08 [1] RSPM (R 3.6.0)                           
 fftwtools         0.9-8    2017-03-25 [1] RSPM (R 3.6.0)                           
 flexdashboard   * 0.5.1.1  2018-06-29 [1] RSPM (R 3.6.0)                                          
 glue              1.4.1    2020-05-13 [1] RSPM (R 3.6.0)                           
 googleAuthR     * 1.3.0    2020-06-16 [1] RSPM (R 3.6.3)                           
 gtable            0.3.0    2019-03-25 [1] RSPM (R 3.6.0)                           
 haven             2.2.0    2019-11-08 [1] RSPM (R 3.6.0)                           
 hms               0.5.3    2020-01-08 [1] RSPM (R 3.6.0)                           
 htmltools         0.5.0    2020-06-16 [1] RSPM (R 3.6.0)                           
 htmlwidgets       1.5.1    2019-10-08 [1] RSPM (R 3.6.0)                           
 httpuv            1.5.2    2019-09-11 [1] RSPM (R 3.6.0)                           
 httr              1.4.1    2019-08-05 [1] RSPM (R 3.6.0)                           
 janitor         * 2.0.1    2020-04-12 [1] RSPM (R 3.6.0)                           
 jpeg              0.1-8.1  2019-10-24 [1] RSPM (R 3.6.0)                           
 jsonlite        * 1.7.0    2020-06-25 [1] CRAN (R 3.6.3)                                         
 leaflet         * 2.0.3    2019-11-16 [1] RSPM (R 3.6.3)                           
 lifecycle         0.2.0    2020-03-06 [1] RSPM (R 3.6.0)                           
 locfit            1.5-9.1  2013-04-20 [1] RSPM (R 3.6.0)                           
 lubridate         1.7.4    2018-04-11 [1] RSPM (R 3.6.0)                           
 magick          * 2.4.0    2020-06-23 [1] RSPM (R 3.6.3)                           
 magrittr          1.5      2014-11-22 [1] RSPM (R 3.6.0)                           
 memoise           1.1.0    2017-04-21 [1] RSPM (R 3.6.0)                           
 mime              0.9      2020-02-04 [1] RSPM (R 3.6.0)                           
 modelr            0.1.5    2019-08-08 [1] RSPM (R 3.6.0)                           
 munsell           0.5.0    2018-06-12 [1] RSPM (R 3.6.0)                           
 nlme              3.1-144  2020-02-06 [2] CRAN (R 3.6.3)                           
 packrat           0.5.0    2018-11-14 [1] RSPM (R 3.6.0)                                  
 pkgbuild          1.0.8    2020-05-07 [1] RSPM (R 3.6.0)                           
 pkgconfig         2.0.3    2019-09-22 [1] RSPM (R 3.6.0)                           
 pkgload           1.1.0    2020-05-29 [1] RSPM (R 3.6.0)                         
 png               0.1-8    2020-01-17 [1] local                                    
 prettyunits       1.1.1    2020-01-24 [1] RSPM (R 3.6.0)                           
 processx          3.4.2    2020-02-09 [1] RSPM (R 3.6.0)                           
 promises          1.1.0    2019-10-04 [1] RSPM (R 3.6.0)                           
 ps                1.3.3    2020-05-08 [1] RSPM (R 3.6.0)                           
 purrr           * 0.3.3    2019-10-18 [1] RSPM (R 3.6.0)                           
 R6                2.4.1    2019-11-12 [1] RSPM (R 3.6.0)                           
 Rcpp              1.0.3    2019-11-08 [1] RSPM (R 3.6.0)                           
 RCurl             1.98-1.2 2020-04-18 [1] RSPM (R 3.6.3)                           
 readr           * 1.3.1    2018-12-21 [1] CRAN (R 3.5.0)                           
 readxl            1.3.1    2019-03-13 [1] RSPM (R 3.6.0)                           
 remotes           2.1.1    2020-02-15 [1] RSPM (R 3.6.0)                           
 reprex            0.3.0    2019-05-16 [1] RSPM (R 3.6.0)                           
 rlang             0.4.6    2020-05-02 [1] RSPM (R 3.6.0)                           
 rmarkdown         2.0      2019-12-12 [1] RSPM (R 3.6.0)                           
 RoogleVision    * 0.0.1.1  2020-07-03 [1] Github (shaun-jacks/RoogleVision@6c2909d)
 rprojroot         1.3-2    2018-01-03 [1] RSPM (R 3.6.0)                           
 rsconnect         0.8.16   2019-12-13 [1] RSPM (R 3.6.0)                           
 rstudioapi        0.11     2020-02-07 [1] RSPM (R 3.6.0)                                              
 sessioninfo       1.1.1    2018-11-05 [1] RSPM (R 3.6.0)                           
 shiny           * 1.5.0    2020-06-23 [1] CRAN (R 3.6.3)                           
 shinycssloaders * 0.3      2020-01-16 [1] RSPM (R 3.6.3)                           
 shinydashboard    0.7.1    2018-10-17 [1] RSPM (R 3.6.3)                           

>[1] /home/R/x86_64-pc-linux-gnu-library/3.6
[2] /opt/R/3.6.3/lib/R/library
Warning message:
In system("timedatectl", intern = TRUE) :
  running command 'timedatectl' had status 1


**I badly need experts' suggestions on this. Kindly help me with this issue.**
